CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Scalyr Agent 2 Changes By Release
 =================================
 
+## 2.1.38 "TBD" - October 17, 2022
+<!---
+Packaged by Joseph Makar <arthurk@sentinelone.com> on Oct 17, 2022 12:31 -0800
+--->
+
+Kubernetes:
+* Fix bug that caused logging of the Kubernetes cache stats to agent status. 
```

Comment:
Good catch, thanks. 

We had a bunch of similar issues in the past (logs leaking status output which can break json parsing). We do have tests for it, but it's almost impossible to test all scenarios since if some log message is logged can depend on config option and environment (we would need to test all the possible permutations).